### DISCUSSION TITLE
How do I use global xp instead of per-guild xp?

### DISCUSSION THREAD

**Discussion by TechPro424** [MrAugu/discord-xp#53] [Q&A] [answered]
How do I use global xp instead of per-guild xp? Like, it should be common across all servers

**Comment by bgencacat** [comment]
Just set guildId hardcoded like 1

`Levels.appendXp(message.author.id, 1, xpAmount);`

**Comment by Myst82015** [comment, marked as answer]
So basically you're right.

The package does **not** validate the guild id meaning you can provide anyhing from a number to a string.

E.g: 
```js
Levels.appendXp(message.author.id, 1, xpAmount);
Levels.appendXp(message.author.id, "1234", xpAmount);
Levels.appendXp(message.author.id, 'global', xpAmount);
```
I prefer 'global' since it actually makes more sense then just 1.

**Reply by TechPro424** [reply]
Got this answer in Discord already, but thanks! (And yeah, I prefer global as well.) : )
